assets/designer/environments/designer-cv-transform/context/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:latest
```

Comment:
use template to resolve immutable date based tag during the build, see other examples in the repo